Config.md

```diff
@@ -64,29 +64,67 @@ with nxp_config(n_jobs=7, verbose=0):
+
+with joblib.parallel_config(n_jobs=4, verbose=55):
+    # NetworkX config for nx-parallel
+    # backend="loky", n_jobs=6, verbose=0
+    nx.square_clustering(G, backend="parallel")
+
+    # Joblib config for other parallel tasks
+    # backend="threading", n_jobs=4, verbose=55
+    joblib.Parallel()(joblib.delayed(sqrt)(i**2) for i in range(10))
+```
+
+- **NetworkX Configurations for nx-parallel**: When calling functions within `nx-parallel`, NetworkX’s configurations will override those specified by Joblib. For example, the `nx.square_clustering` function will use the `n_jobs=6` setting from `nx.config.backends.parallel`, regardless of any Joblib settings within the same context.
+
+- **Joblib Configurations for Other Code**: For any other parallel code outside of `nx-parallel`, such as a direct call to `joblib.Parallel`, the configurations specified within the Joblib context will be applied.
+
+This behavior ensures that `nx-parallel` functions consistently use NetworkX’s settings when enabled, while still allowing Joblib configurations to apply to non-NetworkX parallel tasks.
+
+**Key Takeaway**: When both systems are used together, NetworkX's configuration (`nx.config.backends.parallel`) takes precedence for `nx-parallel` functions. To avoid unexpected behavior, ensure that the `active` setting aligns with your intended configuration system.
+
+### 3.2 Key Differences
+
+- **Parameter Handling**: The main difference is how `backend_params` are passed. Since, in networkx configurations are stored as a [`@dataclass`](https://docs.python.org/3/library/dataclasses.html), we need to pass them as a dictionary, whereas in `joblib.parallel_config` you can just pass them along with the other configurations, as shown below:
+
+  ```py
+  nx.config.backends.parallel.backend_params = {"max_nbytes": None}
+  joblib.parallel_config(backend="loky", max_nbytes=None)
+  ```
 
-- **Usage**: The main difference is how `backend_params` are passed. In `nx.config` you need to pass them as a dictionary, whereas in `joblib.parallel_config` you can just pass them along with the other configurations.
-- **Default Settings**: By default, `nx-parallel` looks for configs set by `joblib.parallel_config`, and NetworkX's config system needs to be enabled explicitly by making the `nx.config.backends.parallel.active` configuration `True`.
+- **Default Behavior**: By default, `nx-parallel` looks for configs in `joblib.parallel_config` unless `nx.config.backends.parallel.active` is set to `True`.
 
-### 3.2 When Should You Use Which System?
+### 3.3 When Should You Use Which System?
 
-- Use NetworkX's config system if you're working with multiple backends within NetworkX.
-- If you're using `nx-parallel` independently, you can choose either system based on your preference.
+- When you're working with multiple NetworkX backends, then to avoid having any conflicts with other backends' code, using networkx to configure nx-parallel would be well suited.
```

Comment:
I think you have mentioned potential conflicts before. What are the potential conflicts?

Reply:
I have updated the `Config.md` with an example of this. Thanks!